scripts/deploy_proxy.sh

```diff
@@ -7,8 +7,10 @@ set -o nounset errexit pipefail
 
 mkdir -p dist
 rm -rf dist/apiproxy
-cp -R apiproxy/ dist/
-sed -i "s/PROXY_BASE_PATH/$APIGEE_BASE_PATH/g" dist/apiproxy/proxies/default.xml
+cp -Rv apiproxy dist/
+sed "s/PROXY_BASE_PATH/$APIGEE_BASE_PATH/g" dist/apiproxy/proxies/default.xml > dist/apiproxy/proxies/default_temp.xml
+cp -f dist/apiproxy/proxies/default_temp.xml dist/apiproxy/proxies/default.xml
+rm -rf dist/apiproxy/proxies/default_temp.xml
 mkdir -p dist/apiproxy/resources/hosted
 rsync -av --copy-links --exclude="node_modules" sandbox/ dist/apiproxy/resources/hosted
```

Comment:
can you please update this rsync command to include:
`--filter=':- .gitignore'`